pkg/reconciler/openshift/tektontrigger/extension.go

```diff
@@ -32,11 +34,20 @@ func OpenShiftExtension(ctx context.Context) common.Extension {
 type openshiftExtension struct{}
 
 func (oe openshiftExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Transformer {
+	type triggersProperties struct {
```

Comment:
@savitaashture IMO, adding some comments on top of this block will help to understand easily, why we need this change

Reply:
Yes make sense 
added comment